django/core/management/commands/squashmigrations.py

```diff
@@ -157,10 +151,7 @@ def handle(self, **options):
         # need to feed their replaces into ours
         replaces = []
         for migration in migrations_to_squash:
-            if migration.replaces:
-                replaces.extend(migration.replaces)
-            else:
-                replaces.append((migration.app_label, migration.name))
+            replaces.append((migration.app_label, migration.name))
```

Comment:
Since above we end up failing when a migration gets `replace`d, I don't believe we could hit this `if` branch (in any case, here I want to point to the squashed migration, as that is where I get the operations from).

Reply:
Thanks, I believe you are correct that we cannot execute the `if` branch today. That `if` branch predates the short-circuiting introduced in ticket-23090.